sdk/metricsadvisor/azure-ai-metricsadvisor/src/main/java/com/azure/ai/metricsadvisor/MetricsAdvisorAsyncClient.java

```diff
@@ -104,9 +104,50 @@
 import static com.azure.core.util.FluxUtil.withContext;
 
 /**
- * This class provides an asynchronous client that contains all the operations that apply to Azure Metrics Advisor.
+ * <p>This class provides an asynchronous client to connect to the Metrics Advisor Azure Cognitive Service.</p>
+ * <p>This client provides asynchronous methods to perform:</p>
+ * <ol>
+ *     <li>Analyze root cause into specific dimension using the
+ *     {@link com.azure.ai.metricsadvisor.MetricsAdvisorClient#listIncidentRootCauses(String, String, Context)}
+ *     method with your respective data source.</li>
+ *     <li>Fetch incidents triggered for a particular detection configuration using the
+ *     {@link com.azure.ai.metricsadvisor.MetricsAdvisorClient#listIncidentsForDetectionConfig(String, OffsetDateTime, OffsetDateTime, ListIncidentsDetectedOptions, Context)}
+ *     method.</li>
+ *     <li>Fetch all the anomalies detected for a particular detection configuration using the
+ *     {@link com.azure.ai.metricsadvisor.MetricsAdvisorClient#listAnomaliesForDetectionConfig(String, OffsetDateTime, OffsetDateTime)}
+ *     method./li>
+ * </ol>
+ *
+ * <p>Service clients are the point of interaction for developers to use Azure Metrics Advisor.
+ * {@link com.azure.ai.metricsadvisor.MetricsAdvisorClient} is the synchronous service client and
+ * {@link com.azure.ai.metricsadvisor.MetricsAdvisorAsyncClient} is the asynchronous service client.
+ * The examples shown in this document use a credential object named DefaultAzureCredential for authentication, which is
```

Comment:
Below we have a sample that also shows sample with `MetricsAdvisorKeyCredential`, we can consider mentioning it here.